Stripe/STPAddress.m

```diff
@@ -202,6 +202,14 @@ - (BOOL)containsRequiredShippingAddressFields:(PKAddressField)requiredFields {
     return containsFields;
 }
 
+- (BOOL)containsRequiredSEPADebitFields {
+    return (self.line1.length > 0
+            && self.city.length > 0
+            && self.country.length > 0
+            && [STPPostalCodeValidator stringIsValidPostalCode:self.postalCode
```

Comment:
Looked into this, and it turns out all SEPA countries do have postal codes (Ireland has postal codes as of 2014: https://en.wikipedia.org/wiki/Postal_addresses_in_the_Republic_of_Ireland). I'll make a PR to remove IE from `countriesWithNoPostalCodes`.